fetch.bs

```diff
@@ -4593,6 +4594,35 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    using <var>request</var>.
   </ol>
 
+ <li>
+  <p>If <var>response</var>'s <a for=response>status</a> is 421, then:
+
+  <ol>
+   <li>
+    <p>If the ongoing fetch is <a for=fetch>terminated</a>, then:
+
+    <ol>
+     <li><p>Let <var>aborted</var> be the termination's aborted flag.
+
+     <li><p>If <var>aborted</var> is set, then return an <a>aborted network error</a>.
+
+     <li><p>Return a <a>network error</a>.
+    </ol>
+
+   <li>
+    <p>If <var>actualResponse</var>'s <a for=response>status</a> is 421,
+     <var>request</var>'s <a for=request>body</a> is non-null, and
+     <var>request</var>'s <a for=request>body</a>'s <a for=body>source</a> is
+      null, then return a <a>network error</a>.
+
+   <li>
+     <p>Set <var>response</var> to the result of performing an <a>HTTP-network-or-cache fetch</a>
+      using <var>request</var> on a new <a>connection</a>, which
+      is <a lt="obtain a connection">obtained</a> using true
+      <var>bypassConnectionPool</var>.
+      User agents should have an <a>implementation-defined</a> recursion limit.
```

Comment:
Similar to the above this needs to be a new argument to HTTP-network-or-cache fetch instead, right? So that it does the right thing when a connection is established.

Reply:
Do you mean adding `bypassConnectionPool` argument to HTTP-network-or-cache fetch and HTTP-network fetch ?

Reply:
Right, yeah. Let me know if you want help rebasing this btw.

Reply:
Done.